src/Pelagos/Bundle/AppBundle/Controller/SecurityController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Pelagos\Bundle\AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+
```

Comment:
Can remove extra lines between use statements

Reply:
Normally we separate use statements by namespace.